dags/springer/dag_pull_ftp.py

```diff
@@ -5,22 +5,27 @@
 from common.sftp_service import SFTPService
 from springer.repository import SpringerRepository
 from springer.sftp_service import SpringerSFTPService
+from structlog import get_logger
 
 
 @dag(start_date=airflow.utils.dates.days_ago(0))
 def springer_pull_ftp():
+    logger = get_logger().bind(class_name="springer_pull_ftp")
+
     @task()
     def migrate_from_ftp(
         sftp: SFTPService = SpringerSFTPService(),
         repo: IRepository = SpringerRepository(),
+        **kwargs
     ):
-        pull_ftp.migrate_from_ftp(sftp, repo)
+        return pull_ftp.migrate_from_ftp(sftp, repo, logger, **kwargs)
 
     @task()
-    def trigger_file_processing(repo: IRepository = SpringerRepository()):
-        return pull_ftp.trigger_file_processing("springer", repo)
+    def trigger_file_processing(filenames=[], repo: IRepository = SpringerRepository()):
```

Comment:
it's not a good practise to have mutable args in python https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

Reply:
Ohh yes I remember thtat 🥲